ynr/apps/bulk_adding/tests/test_bulk_add.py

```diff
@@ -81,7 +82,7 @@ def test_submitting_form(self):
         # make it lower and at least make sure it's not getting bigger.
         #
         # [1]: https://github.com/DemocracyClub/yournextrepresentative/pull/467#discussion_r179186705
-        with self.assertNumQueries(56):
+        with self.assertNumQueries(FuzzyInt(56, 59)):
```

Comment:
I have added comments below at where the increases have come from

Reply:
Thanks for thinking about this! Although this number is growing, I'm not so fussed about it growing a small amount. This check is really to make sure we've not forgotten a `select_related` somewhere